e2e/testsuite/testconfig.go

```diff
@@ -741,7 +741,10 @@ func modifyChannelGenesisAppState(ibcAppState []byte) ([]byte, error) {
 	}
 
 	// be ashamed, be very ashamed
-	channelGenesis := ibcGenesisMap["channel_genesis"].(map[string]interface{})
+	channelGenesis, ok := ibcGenesisMap["channel_genesis"].(map[string]interface{})
+	if !ok {
+		return nil, fmt.Errorf("can't convert ibcGenesisMap entry into map[string]interface{} type")
```

Comment:
For this kind of errors we would usually do something like...

```suggestion
		return nil, fmt.Errorf("can't convert IBC genesis map entry into type %T", &map[string]interface{})
```

Reply:
Done, though I had to do a small workaround as it won't work by putting directly `map[string]interface{}` there

Error:
```
codec.go:1: : # github.com/cosmos/ibc-go/e2e/testsuite
./testconfig.go:746:79: map[string]interface{} (type) is not an expression (typecheck)
```